### PR TITLE
[16.0][MIG] event_registration_multi_qty: Migration to version 16.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT43666

@chienandalu please review!

### CODE REVIEW

File: event_registration_multi_qty/models/event.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            event.update(vals)
+        rest = self - multi_qty_events
+        return super(EventEvent, rest)._compute_seats()
+
+    @api.constrains("registration_multi_qty")
+    def _check_attendees_qty(self):
+        for event in self:
+            if (
+                not event.registration_multi_qty
+                and max(event.registration_ids.mapped("qty") or [0]) > 1
```
Comment (chienandalu) [inline event_registration_multi_qty/models/event.py]: Better:
```suggestion
                and max(event.registration_ids.mapped("qty"), default=0) > 1
```

**Review by chienandalu — APPROVED** [review]
Test and code review  :+1: . Tiny suggestion:

File: event_registration_multi_qty/models/event.py (hunk 1 above)
Comment (stefan-tecnativa) [reply event_registration_multi_qty/models/event.py]: Done, thanks for the suggestion.

**Review by chienandalu — APPROVED** [review]

**Comment by chienandalu** [comment]
please review @CarlosRoca13

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**